.gitignore

```diff
@@ -19,3 +19,5 @@ doc-source/*.rst
 *.log
 *.iml
 .idea/
+/cwmsSystemJars/
```

Comment:
This directory shouldn't even exist anymore. if it's still on your system you can just delete it.